api/pkg/request/version_check_handler.go

```diff
@@ -14,7 +14,7 @@ var (
 
 func init() {
 	MinimumVersions = map[string]string{
-		"happy-cli":      "0.53.6",
+		"happy-cli":      "0.90.0",
```

Comment:
blocks all requests from CLI versions below 0.90.0, returning a message that the user should upgrade happy